cron/main.go

```diff
@@ -0,0 +1,33 @@
+package cron
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+const (
+	host	= "trotter.cyverse.org"
+	port   	= 6432
+	user   	= "de"
+	dbname 	= "de"
+	folder  = "/tmp"
+	logfile = "logs-stdout-output"
+)
```

Comment:
I may have mentioned this before, but an upcoming task would be to move these configuration settings outside of the source code. The `permissions` service reads the values from a YAML configuration file, which is one option. Another option is to use environment variables, which is what a lot of the VICE services do. This can be done in a future PR without a problem; I just wanted to mention it now because these lines of code won't show up in future PR diffs unless they're modified.

Reply:
Noted. I think next time we sit down we should figure out what to do about this.